Mathlib/MeasureTheory/Function/LpSpace.lean

```diff
@@ -752,6 +753,26 @@ protected lemma Memℒp.piecewise [DecidablePred (· ∈ s)] {g}
 
 end Indicator
 
+section Topology
+variable {X : Type*} [TopologicalSpace X] [MeasurableSpace X] [OpensMeasurableSpace X]
```

Comment:
Could you move the assumption `[OpensMeasurableSpace X]` to the theorem that needs it, i.e., the second one.